subprojects/docs/src/docs/release/notes.md

```diff
@@ -77,6 +77,11 @@ that take such nested zips as an input, e.g. when testing applications packaged
 The ZIP analysis now also avoids unpacking entries that are irrelevant, e.g. resource files on a compile classpath. 
 This improves performance for projects with a large amount of resource files.
 
+## File case changes are handled for `copy` and `sync` tasks
+
+On case insensitive file systems (e.g. `Windows` and `Mac`), a file/folder rename (where only the case is changed) will now be detected properly in `copy` and related tasks (e.g. `sync`).
+The task's subsequent run will not be up-to-date and won't truncate the output anymore.
```

Comment:
We usually do not describe bugs fixed here, as they are mentioned in the list of closed issues. Let's skip this sentence.

Reply:
Done